parser/Cargo.toml

```diff
@@ -23,6 +23,7 @@ unic-ucd-ident  = "0.9.0"
 unicode_names2 = "0.5.0"
 phf = "0.10.1"
 ahash = "0.7.6"
+itertools = "0.10.3"
```

Comment:
(Had to add this to the `parser`, assume that's ok? Also, unclear if you want to keep these sorted.)

Reply:
sorting will be good. this is also ok.